scripts/run-tests.sh

```diff
@@ -55,13 +55,10 @@ run_tests action-token-required-action -Pwildfly-managed
 
 if [ -n "$KEYCLOAK_VERSION" ]; then
   JS_VERSION_OPTION="-Dkeycloak.js.version=$KEYCLOAK_VERSION"
-elif [ -n "$PRODUCT_VERSION" ]; then
-  JS_VERSION_OPTION="-Dkeycloak.js.version=$PRODUCT_VERSION"
```

Comment:
Just a quick question. We'll be using `KEYCLOAK_VERSION` even when testing prod?